### DISCUSSION TITLE
I think we should have a wiki

### DISCUSSION THREAD

**Discussion by sonakravtsenko3** [thygate/stable-diffusion-webui-depthmap-script#293] [General]
There are many things you can do with this repo, I think it would be a good idea to start GitHub Wiki and list all the things there in different articles.
https://github.com/godotengine/godot/issues/2019#issuecomment-107373724

**Comment by sonakravtsenko3** [comment]
Things that I think should go to the wiki:
Changelog (except maybe some of the most recent, maybe version only?)
Forks and Related
Credits to contributors.

I think we are kinda obliged to leave Acknowledgements on the front page - the authors did the heavy lifting part (the research) and it should be easy to find the original source. It should be obvious that this repository works only thanks to these.

The main page is somewhat outdated overall... I think it would be reasonable to update it after #290.

**Reply by thygate** [reply]
Yeah I've been wanting to get rid of the changelog in the readme for a while, and I agree with keeping the acknowledgements. 
I had considered a wiki a few times, but did not get around to it, it is definitely a good idea as there is a lot of info to share, and the readme is a trainwreck ..

**Comment by graemeniedermayer** [comment]
So for wiki editing. It sounds like that only for colloborators? https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis 

Maybe I missing something because that doesn't sound very wiki-like.

**Reply by sonakravtsenko3** [reply]
@thygate I think this is a good idea to allow public editing. If you don't mind, could you please unselect "Restrict editing to collaborators only"? There is possibility that we will need to change this back at some point... But hopefully this does not happen.
